zebra-consensus/src/block/subsidy/general.rs

```diff
@@ -118,10 +118,44 @@ pub fn output_amounts(transaction: &Transaction) -> HashSet<Amount<NonNegative>>
         .collect()
 }
 
+/// Lockbox funding stream total input value for a block height.
+///
+/// Assumes a constant funding stream amount per block.
+// TODO: Cache the lockbox value balance in zebra-state (will be required for tracking lockbox
+//       value balance after ZSF ZIPs or after a ZIP for spending from the deferred pool)
```

Comment:
What `ZSF ZIPs` means ?

Reply:
Ah, sorry, it was referring to the Zcash Sustainability Fund ZIPs, updated in https://github.com/ZcashFoundation/zebra/pull/8694/commits/bf4943270366430d218fb6a4900fc5ab76aba7b3.

It's specifically [ZIP 234](https://github.com/zcash/zips/pull/706) that will cause an issue, we can probably remove this function altogether in https://github.com/ZcashFoundation/zebra/pull/8729.